components/sensor/ld2410.rst

```diff
@@ -96,86 +298,445 @@ and binary sensors.
+        out_pin_level:
+          name: out pin level
+
+.. _ld2410-light-function:
+
+Configuration variables:
+************************
+
+- **distance_resolution** (*Optional*): Control the gates distance resolution. Can be ``0.75m`` or ``0.2m``. Defaults to ``0.75m``.
+  All options from :ref:`Select <config-select>`.
+- **baud_rate** (*Optional*): Control the serial port baud rate. Defaults to ``256000``.
+  Once changed, all sensors will stop working until a fresh install with an updated :ref:`UART Component <uart>` configuration.
+  All options from :ref:`Select <config-select>`.
+- **light_function** (*Optional*): If set, will affect the OUT pin value, based on :ref:`light threshold<ld2410-light-threshold>`. Can be ``off``, ``low`` or ``above``. Defaults to ``off``.
+  All options from :ref:`Select <config-select>`.
+- **out_pin_level** (*Optional*): Control OUT pin ``away`` value. Can be ``low`` or ``high``. Defaults to ``low``.
+  All options from :ref:`Select <config-select>`.
+- **ld2410_id** (*Optional*, :ref:`config-id`): Manually specify the ID for the :doc:`ld2410` component if you are using multiple components.
+
+Automations
+-----------
+
+``bluetooth_password.set`` Action
+*********************************
+
+This is an :ref:`Action <config-action>` for setting the bluetooth password.
+
+.. code-block:: yaml
+
+    - bluetooth_password.set:
+        id: my_ld2410
+        password: "HiLink"
+
+Configuration variables:
+
+- **id** (**Required**, :ref:`config-id`): The ID of the :doc:`ld2410` component to set.
+- **password** (**Required**, string, :ref:`templatable <config-templatable>`):
+  The password to set. Case sensitive. Must be exactly 6 characters long. Default password is `HiLink`.
+
+To change the password from HA you can use the following example config:
+
+.. code-block:: yaml
+
+    ld2410:
+      id: my_ld2410
+
+    api:
+      services:
+        - service: set_ld2410_bluetooth_password
+          variables:
+            password: string
+          then:
+            - bluetooth_password.set:
+                id: my_ld2410
+                password: !lambda 'return password;'
+
+OUT pin
+-------
+
+In order to monitor the presence indicated by the component, with the :ref:`light function<ld2410-light-function>` taken
+under account, you can set up a :ref:`GPIO Binary Sensor <gpio-binary-sensor>`:
+
+.. code-block:: yaml
+
+    binary_sensor:
+      - platform: gpio
+        pin: REPLACE ME
+        name: gpio out pin presence
+        device_class: presence
+
+
+Calibration Process
+-------------------
+
+In order to calibrate your ``ld2410`` sensor perform the following:
+
+1. Enable :ref:`engineering mode<ld2410-engineering-mode>`.
+2. Monitor the ``gX_move_energy`` and ``gX_still_energy`` :ref:`sensors<ld2410-sensors>`.
+3. Change the :ref:`thresholds<ld2410-number>` and repeat step 2 until satisfaction.
+4. Disable :ref:`engineering mode<ld2410-engineering-mode>`.
+
+Home Assistant Card
```

Comment:
Please remove this section. ESPHome should not be recommending custom cards etc on Home Assistant.

Reply:
Ok, where should we recommend it, it is a very elaborate card as there are a lot of entities produced by the integration

Reply:
fixed - i did a card that doesn't require external custom cards